### PR TITLE
Separate baseline files into parameter files and expected results files

### PR DESCRIPTION
**Pull request by becantwell**

### CODE REVIEW

File: test/pytest/test_xpedite/data/allocatorAppRegularManifest.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+file name, size, lines
+allocatorAppRegular/__init__.py,2,1
```
Comment (Mani-D) [inline test/pytest/test_xpedite/data/allocatorAppRegularManifest.csv]: do we need __init__.py ?

File: test/pytest/test_xpedite/data/dataTxnAppRegularManifest.csv [hunk 2]
```diff
@@ -0,0 +1,10 @@
+file name, size, lines
+dataTxnAppRegular/__init__.py,0,0
```
Comment (Mani-D) [inline test/pytest/test_xpedite/data/dataTxnAppRegularManifest.csv]: `__init__.py` here as well. is it needed ?

File: test/pytest/test_xpedite/test_profiler/profile.py [hunk 3]
```diff
@@ -31,8 +39,9 @@ def runXpediteReport(runId, context, scenario, sampleFilePath=None, cpuInfoOverr
   with scenario.makeXpediteDormantApp(runId, context.workspace, sampleFilePath) as xpediteApp:
     if cpuInfoOverride:
       xpediteApp.env.proxy.fullCpuInfo = scenario.fullCpuInfo
-    xpediteApp.appInfoPath = os.path.join(scenario.dataDir, XPEDITE_APP_INFO_PATH)
-    return generateProfiles(xpediteApp, scenario)
+    xpediteApp.appInfoPath = os.path.join(scenario.dataDir, XPEDITE_APP_INFO_PARAMETER_PATH)
+    profiles = generateProfiles(xpediteApp, scenario)
```
Comment (Mani-D) [inline test/pytest/test_xpedite/test_profiler/profile.py]: unnecessary assignment to varaible `profiles`

File: test/pytest/test_xpedite/test_profiler/test_profiler.py [hunk 4]
```diff
@@ -58,7 +59,8 @@ def test_report_vs_baseline():
   """
   for scenarios in SCENARIO_LOADER:
     with scenarios as scenarios:
-      compareVsBaseline(CONTEXT, scenarios)
+      pass # only load scenarios
```
Comment (Mani-D) [inline test/pytest/test_xpedite/test_profiler/test_profiler.py]: is this intended ? why not call compareVsBaseline in the scope of with ?

File: test/pytest/test_xpedite/test_profiler/test_profiler.py [hunk 5]
```diff
@@ -100,10 +105,10 @@ def test_probe_states(capsys):
         probes = loadProbes(CONTEXT, scenarios)
         for probe in probes:
           probeMap[probe.sysName] = probe
-      assert len(probes) == len(scenarios.baselineProbeMap.keys())
-      findDiff(probeMap, scenarios.baselineProbeMap)
-      for probe in probeMap.keys():
-        assert compareProbes(probeMap[probe], scenarios.baselineProbeMap[probe])
+    assert len(probes) == len(scenarios.baselineProbeMap.keys())
```
Comment (Mani-D) [inline test/pytest/test_xpedite/test_profiler/test_profiler.py]: why move these outside with scope ?

File: test/runTest.sh [hunk 6]
```diff
@@ -117,6 +117,8 @@ function runPytests() {
 
   PYTHONPATH=${XPEDITE_DIR}:${PYTHONPATH} pytest ${COV} ${TEST_NAME} -v ${APP_HOST} ${TRANSACTION_COUNT} ${THREAD_COUNT} ${WORKSPACE} ${RUN_DIR_ARG} ${APPS}
 
+  ${TEST_DIR}/validateTarFiles.sh
```
Comment (Mani-D) [inline test/runTest.sh]: running test should not validate tar files. Why is this added ?

Comment (becantwell) [reply test/runTest.sh]: I think these are issues from the merge, I will fix them.

File: test/pytest/test_xpedite/data/dataTxnAppRegularManifest.csv (hunk 2 above)
Comment (becantwell) [reply test/pytest/test_xpedite/data/dataTxnAppRegularManifest.csv]: I think so, to import profileInfo from the regular scenario in the benchmark scenario
from dataTxnAppRegular.parameters.profileInfo import *

File: test/runTest.sh (hunk 6 above)
Comment (becantwell) [reply test/runTest.sh]: I put this here because the check is in generateBaseline.sh, but that script isn't run during the automated test. I can get rid of it.
